app/controllers/verify/usps_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Verify
+  class UspsController < ApplicationController
+    include IdvStepConcern
+
+    before_action :confirm_step_needed
+
+    def index
+      @applicant = idv_session.resolution.vendor_resp.normalized_applicant
+    end
+
+    def create
+      idv_session.address_verification_mechanism = :usps
+      redirect_to verify_review_url
```

Comment:
Interesting!  I never knew about this HTTP spec but seems 100% sensible. Will be doing this from now on!